src/Employer/Employer.Web/Orchestrators/VacancyViewOrchestrator.cs

```diff
@@ -72,6 +72,12 @@ public async Task<DisplayVacancyViewModel> GetFullVacancyDisplayViewModelAsync(V
                     await _vacancyDisplayMapper.MapFromVacancyAsync(submittedViewModel, vacancy);
                     submittedViewModel.SubmittedDate = vacancy.SubmittedDate.Value.AsGdsDate();
                     return submittedViewModel;
+                case VacancyStatus.Review:
+                    var reviewViewModel = new ReviewVacancyViewModel();
+                    reviewViewModel.EducationLevelName =
```

Comment:
this property set could be within the initialisation of the object:

Reply:
done